src/screens/LoginPage/LoginPage.tsx

```
@@ -279,44 +300,50 @@
+        {
+          onRequest: () => {
+            setLoginLoading(true);
+          },
+          onResponse() {
+            setLoginLoading(false);
+          },
+        },
+      )) as {
+        data: IAuthResponse | null;
+      };
+      if (signInData && signInData.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)